configuration/env_var_processors.rst

```diff
@@ -232,8 +232,8 @@ Symfony provides the following env var processors:
             ]);
 
 ``env(resolve:FOO)``
-    Replaces the string ``FOO`` by the value of a config parameter with the
-    same name:
+    Replace all matches of ``%param_name%`` in the parameter ``FOO`` by the values of a config parameters with the
```

Comment:
I agree, the description here is hard to understand and your change makes it better, but I still find this a bit hard to understand, how about:

> If the value of the parameter ``FOO`` contains other parameters, they will be replaced with their corresponding values.

Reply:
I liked this proposal better ... but it had some issues too (e.g. `FOO` is not called "a parameter" in the rest of the article, etc.)

So, we reworded it as follows:

```rst
    If the content of ``FOO`` includes container parameters (with the syntax
    ``%parameter_name%``), it replaces the parameters by their values:
```